app/uk/gov/hmrc/apinotificationpull/controllers/NotificationsController.scala

```diff
@@ -51,7 +51,9 @@ class NotificationsController @Inject()(apiNotificationQueueService: ApiNotifica
     implicit val hc: HeaderCarrier = buildHeaderCarrier(request)
 
```

Comment:
Not really related to this PR but something that I missed in a previous one:
In `NotificationsController` we have 2 ways of passing implicit `HeaderCarrier` for getAll and Delete. Can we make it consistent for both.